zwave_js_server/model/statistics.py

```diff
@@ -75,3 +75,15 @@ def route_failed_between(self) -> Optional[List["Node"]]:
             return None
         assert self.client.driver
         return [self.client.driver.controller.nodes[node_id] for node_id in node_ids]
+
+    def as_dict(self) -> Dict[str, Optional[Union[List[int], int]]]:
```

Comment:
I'd use a TypedDict to type the return value.

Reply:
good idea